code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -189,3 +189,72 @@
 		rad = 0
 	)
 	price_tag = 500
+
+/obj/item/clothing/gloves/dusters
+	name = "steel knuckle dusters"
+	desc = "More pain for them, less for you."
+	description_info = "Have to be worn in your gloves slot to increase your punch damage."
+	icon_state = "dusters"
+	item_state = "dusters"
+	var/punch_increase = 5
+	price_tag = 20
+
+/obj/item/clothing/gloves/dusters/silver
+	name = "silver knuckle dusters"
+	desc = "More pain for them, more bling for you."
+	icon_state = "dusters_silver"
+	item_state = "dusters_silver"
+	price_tag = 40
+	style = STYLE_LOW
+
+/obj/item/clothing/gloves/dusters/plasteel
+	name = "plasteel knuckle dusters"
+	desc = "More pain for them, now doubled."
+	icon_state = "dusters_plasteel"
+	item_state = "dusters_plasteel"
+	punch_increase = 10
+	price_tag = 60
+
+/obj/item/clothing/gloves/dusters/gold
+	name = "golden knuckle dusters"
+	desc = "More pain for them, more style for you."
+	icon_state = "dusters_gold"
+	item_state = "dusters_gold"
+	punch_increase = 10
+	price_tag = 100
+	style = STYLE_HIGH
+
+/obj/item/clothing/gloves/dusters/platinum
+	name = "spiked platinum knuckle dusters"
+	desc = "Hurt like hell, and stylish as well."
+	icon_state = "dusters_platinum"
+	item_state = "dusters_platinum"
+	punch_increase = 15
+	price_tag = 120
+	style = STYLE_HIGH
+
+/obj/item/clothing/gloves/dusters/gloves
+	name = "knuckle gloves"
+	desc = "Gloves with additional reinforcment on the knuckles. \
+	These have plasteel powder sewn into the knuckles, adding more kinetic energy to your punches."
+	icon_state = "knuckles"
+	item_state = "knuckles"
+	punch_increase = 10
+	style = STYLE_HIGH
+	armor = list(
+		melee = 4,
+		bullet = 1,
+		energy = 0,
+		bomb = 0,
+		bio = 0,
+		rad = 0
+	)
+	price_tag = 540
+
+/obj/item/clothing/gloves/dusters/equipped(mob/living/carbon/human/H)
+	..()
+	H.punch_damage_increase += punch_increase
+
+/obj/item/clothing/gloves/dusters/dropped(mob/living/carbon/human/H)
+	H.punch_damage_increase -= punch_increase
+	..()
```

Comment:
Do it using COMSIG_CLOTH_DROPPED (triggered when you remove a cloth from yourself) and COMSIG_CLOTH_EQUIPPED.

Also dropped is not the same than unequipped. Here I feel I could reach infinite punch damage by just equipping the knuckles in my hand slot than putting them in my active hand, and again, and again. `equipped` will trigger each time while dropped will not (because I'm not dropping the item, just taking it into my hand).

Reply:
No one punch man?
![image](https://user-images.githubusercontent.com/65828539/193139504-774c5f9c-782e-4a4a-b180-1c3337f2b021.png)

Reply:
![image](https://user-images.githubusercontent.com/64754494/193141158-325c46ee-28f4-412c-a1c2-074e01183ade.png)

Reply:
1984